### ISSUE TITLE
Doesnt work with embercli with ember 2.0

### ISSUE THREAD

**Issue by abhayathapa** [poetic/ember-cli-photoswipe#26]
I created items on model on route and used the {{photoswipe}} as in this dummy app but nothing renders and no error is given

**Comment by lifeart** [comment]
+1

**Comment by dzena** [comment]
I have the same problem. I'm using ember 1.12.0 and ember-cli 1.13.8.
Solve the problem in the following way though:

```
{{#photo-swipe}}
 {{#each model.product.pictures as |img|}}
     <a class="photo-item" href={{img.readUrl}} data-width="100" data-height="100">
        <img src={{img.readUrl}} width="100" height="100" alt={{img.fileName}}>
     </a>
 {{/each}}
{{/photo-swipe}}

```

I just realised that this is def. not a good solution. Mainly because data- attributes will always have a static value, and that fact will affect your pictures (height and width) when you open the gallery.

**Comment by davidgoli** [comment]
Please don't do as @dzena's example shows - this is deprecated, breaks the bindings, and prevents updates to your `model` from being reflected in the slideshow. Instead, make sure your items are structured as follows:

``` javascript
items = [
  {
    src: 'http://your/photo.jpg',
    w: 320,
    h: 480,
    title: 'fancy pic',
    msrc: 'bigger size url'
  }
];
```

Then invoke the component like so:

``` hbs
{{#photo-swipe items=items as |item|}}
  <img src={{item.src}} alt={{item.title}} />
{{/photo-swipe}}
```

For most of us, this will mean mapping your model properties into the PhotoSwipe item structure.

**Comment by davidgoli** [comment]
Oh and here's another thing - the latest published version `1.1.0` is not using the same code as you see on the `master` branch; it's a few commits behind and doesn't work with the `README` you see here. Hopefully the maintainers of this addon will wake up from their deep deep sleep and publish a new version with what they have here in master. Until then, you can pin in `package.json`:

``` json
"ember-cli-photoswipe": "poetic/ember-cli-photoswipe#master"
```

**Comment by dzena** [comment]
@davidgoli 
I still have the same problem. Nothing is being rendered.
ember-cli: 1.13.8
ember:  1.12.0

``` javascript
 items: Ember.computed("model.product.pictures.[]", function () {
    var pictures = this.get("model.product.pictures");

    var items = [];

    pictures.forEach(function (picture) {
      var item = {
        src: picture.get("readUrl"),
        w: 320,
        h: 480,
        title: picture.get("fileName"),
        msrc: "bigger"
      };

      items.push(item);
    });

    return items;
  }),
```

``` handlebars
{{#photo-swipe items=items as |item|}}
      <img class="thumb" src={{item.src}} alt={{item.title}} />
{{/photo-swipe}}
```

**Comment by davidgoli** [comment]
@dzena what version of `ember-cli-photoswipe` are you running?

**Comment by dzena** [comment]
```
"ember-cli-photoswipe": "poetic/ember-cli-photoswipe#master"
```

**Comment by davidgoli** [comment]
I just published `1.2.0` to npm so you should once again be able to point to that version now for this functionality.
